<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>客户端与服务器架构系统-TCP服务器 Version 1.8</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/new/prefix1/images/logo.ico</normaloff>:/new/prefix1/images/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_server">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>服务器监听地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_port">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>服务器监听端口：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboIP">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_StartListening">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>启动监听</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/start.ico</normaloff>:/new/prefix1/images/start.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_StopListening">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>停止监听</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/stop.ico</normaloff>:/new/prefix1/images/stop.ico</iconset>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>101</y>
     <width>711</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="editMsg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>711</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>71</y>
     <width>731</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>消息列表</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>接收文件进度：</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>480</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>309</y>
     <width>731</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>发送消息</string>
   </property>
   <widget class="QPushButton" name="pushButton_ExitSystem">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出系统</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/new/prefix1/images/exit.ico</normaloff>:/new/prefix1/images/exit.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ClearMsg">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>清除消息</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/new/prefix1/images/clear.ico</normaloff>:/new/prefix1/images/clear.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SendData">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>发送消息</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/new/prefix1/images/sendmsg.ico</normaloff>:/new/prefix1/images/sendmsg.ico</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_stateip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>122</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>731</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QSpinBox" name="spinPort">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>6688</number>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>label_server</zorder>
  <zorder>label_port</zorder>
  <zorder>comboIP</zorder>
  <zorder>pushButton_StartListening</zorder>
  <zorder>pushButton_StopListening</zorder>
  <zorder>plainTextEdit</zorder>
  <zorder>editMsg</zorder>
  <zorder>label_3</zorder>
  <zorder>progressBar</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
